integration/cypress/integration/deleteCompletedJobRun.spec.ts

```diff
@@ -0,0 +1,11 @@
+context('End to end', function() {
+  it('Deletes a completed job', () => {
+    cy.login()
+
+    // Create Job
```

Comment:
This seems like it's missing job creation

Reply:
Whoops, that comment was a c/p mistake. We can't actually (easily) test deleting job runs since job runs have to be over a week old before they can be deleted. We'd have to seed the DB with old job runs in order to test this properly, and my understanding was that doing so was a bit overkill for these tests. I added a comment with this explanation for future reference.